src/IoC/HandlerExtensions.cs

```diff
@@ -33,7 +34,9 @@ public static IServiceCollection AddHandlers(this IServiceCollection services)
                 .AddSingleton<IHandler, JsonResultHandler<IEnumerable<StorageSiteResource>>>()
                 .AddSingleton<IHandler, JsonResultHandler<StorageLocationResource>>()
                 .AddSingleton<IHandler, JsonResultHandler<IEnumerable<StorageLocationResource>>>()
-                .AddSingleton<IHandler, JsonResultHandler<IEnumerable<FunctionCodeResource>>>();
+                .AddSingleton<IHandler, JsonResultHandler<IEnumerable<StorageLocationResource>>>()
+                .AddSingleton<IHandler, JsonResultHandler<GoodsInLogEntryResource>>()
```

Comment:
probably dont need this line  - I dont expect we'll ever request a single GoodsInLogEntryResource